src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -63,7 +65,16 @@ object Boot extends App with LazyLogging {
     val cloudExt = googleServicesConfigOption match {
       case Some(googleServicesConfig) =>
         val petServiceAccountConfig = config.as[PetServiceAccountConfig]("petServiceAccount")
-        val googleDirectoryDAO = new HttpGoogleDirectoryDAO(googleServicesConfig.appName, Pem(WorkbenchEmail(googleServicesConfig.serviceAccountClientId), new File(googleServicesConfig.pemFile), Option(googleServicesConfig.subEmail)), "google")
+
+        val googleDirDaos = (if (googleServicesConfig.adminSdkServiceAccounts.isEmpty) {
+          Seq(ServiceAccountConfig(googleServicesConfig.pemFile, googleServicesConfig.serviceAccountClientId))
+        } else {
+          googleServicesConfig.adminSdkServiceAccounts
+        }).map { serviceAccountConfig =>
+          new HttpGoogleDirectoryDAO(googleServicesConfig.appName, Pem(WorkbenchEmail(serviceAccountConfig.serviceAccountClientId), new File(serviceAccountConfig.pemFile), Option(googleServicesConfig.subEmail)), "google")
```

Comment:
shall we remove the previous `pemFile` and `serviceAccountClientId` field, and just have the List of accounts?

Reply:
This change needs to be backwards compatible with current config settings. We need to leave it alone for now.